pkg/operator/bootstrap.go

```diff
@@ -95,7 +105,7 @@ func RenderBootstrap(
 		return fmt.Errorf("expected *configv1.DNS found %T", obji)
 	}
 
-	spec, err := createDiscoveredControllerConfigSpec(infra, network, proxy, dns)
+	spec, err := createDiscoveredControllerConfigSpec(infra, node, network, proxy, dns)
```

Comment:
would be nice to wrap all these variables into a structure to make it easier to extend later